src/FunctionLoader.cs

```diff
@@ -18,6 +18,7 @@ namespace Microsoft.Azure.Functions.PowerShellWorker
     internal class FunctionLoader
     {
         private readonly Dictionary<string, AzFunctionInfo> _loadedFunctions = new Dictionary<string, AzFunctionInfo>();
+        private const string latestAzModulePath = @"D:\Program Files (x86)\ManagedDependencies\PowerShell\AzPSModules\1.0.0";
```

Comment:
nit we've been using underscore to denote it's private.
```suggestion
        private const string _latestAzModulePath = @"D:\Program Files (x86)\ManagedDependencies\PowerShell\AzPSModules\1.0.0";
```

Reply:
I can follow _ pattern for this code base but in C# camel case without _ is normal for class members